app/attributes/attributecontroller.h

```diff
@@ -167,6 +167,7 @@ class  AttributeController : public QObject
     void featureIdChanged();
     void changesCommited();
     void commitFailed();
+    void rawValueChanged();
```

Comment:
What is the point of this signal?

Reply:
Mayber I get it wrong, but we talked about it and have it in the diagrams. Removed.